src/screens/Moderation/index.tsx

```diff
@@ -423,45 +424,52 @@ export function ModerationScreenInner({
-                      ]}>
-                      <LabelingService.Avatar avatar={labeler.creator.avatar} />
-                      <LabelingService.Content>
-                        <LabelingService.Title
-                          value={getLabelingServiceTitle({
-                            displayName: labeler.creator.displayName,
-                            handle: labeler.creator.handle,
-                          })}
-                        />
-                        <LabelingService.Description
-                          value={labeler.creator.description}
-                          handle={labeler.creator.handle}
-                        />
-                      </LabelingService.Content>
-                    </LabelingService.Outer>
-                  )}
-                </LabelingService.Link>
-              </React.Fragment>
+          {labelers
+            .filter(
+              labeler =>
+                !isNonConfigurableModerationAuthority(labeler.creator.did),
             )
-          })}
+            .map((labeler, i) => {
+              return (
+                <React.Fragment key={labeler.creator.did}>
+                  {i !== 0 && <Divider />}
+                  <LabelingService.Link labeler={labeler}>
+                    {state => (
+                      <LabelingService.Outer
+                        style={[
+                          i === 0 && {
+                            borderTopLeftRadius: a.rounded_sm.borderRadius,
+                            borderTopRightRadius: a.rounded_sm.borderRadius,
+                          },
+                          i === labelers.length - 1 && {
+                            borderBottomLeftRadius: a.rounded_sm.borderRadius,
+                            borderBottomRightRadius: a.rounded_sm.borderRadius,
+                          },
+                          (state.hovered || state.pressed) && [
+                            t.atoms.bg_contrast_50,
+                          ],
+                        ]}>
+                        <LabelingService.Avatar
+                          avatar={labeler.creator.avatar}
+                        />
+                        <LabelingService.Content>
+                          <LabelingService.Title
+                            value={getLabelingServiceTitle({
+                              displayName: labeler.creator.displayName,
+                              handle: labeler.creator.handle,
+                            })}
+                          />
+                          <LabelingService.Description
+                            value={labeler.creator.description}
+                            handle={labeler.creator.handle}
+                          />
+                        </LabelingService.Content>
+                      </LabelingService.Outer>
+                    )}
+                  </LabelingService.Link>
+                </React.Fragment>
+              )
+            })}
         </View>
```

Comment:
A suggestion/question: Would it be possible to add a notice to the UI whenever a non-configurable authority is enabled for the current region?

<img width="363" alt="image" src="https://github.com/user-attachments/assets/2145c4a9-b8f9-432d-86bd-9c70eb21043b">

```suggestion
        </View>
      {labelers?.some(
        labeler => !isNonConfigurableModerationAuthority(labeler.creator.did),
      ) && (
        <View
          style={[
            a.p_lg,
            a.rounded_sm,
            t.atoms.bg_contrast_25,
            a.mt_xl,
            a.flex_row,
            a.gap_md,
          ]}>
          <View
            style={[
              a.align_center,
              a.justify_start,
              a.p_md,
              {width: 40},
              // t.atoms.bg_contrast_200,
            ]}>
            <Flag style={t.atoms.text_contrast_medium} />
          </View>
          <Text
            style={[
              a.leading_snug,
              t.atoms.text_contrast_high,
              a.flex_shrink,
              a.pr_sm,
            ]}>
            <Trans>
              In addition to the moderation services configured above, takedown
              requests from local authorities of the current region (
              {device.get(['geolocation'])?.countryCode ?? 'Unknown'}) are
              enabled in compliance with local laws.
            </Trans>
          </Text>
        </View>
      )}
```

Reply:
Thanks for this suggestion! We discussed internally and decided to show the regional labeler when its applied, even though it's non-configurable. That way it's very clear what's happening.

Reply:
Awesome! Thanks for this. It strikes a healthy balance between following local laws and still providing transparency about them.